test/feature/test_lightglue_onnx.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import torch
+
+from kornia.feature import OnnxLightGlue
+
+
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="OnnxLightGlue requires CUDA")
+class TestOnnxLightGlue:
+    @pytest.mark.slow
+    @pytest.mark.parametrize("weights", OnnxLightGlue.MODEL_URLS.keys())
+    def test_pretrained_weights(self, weights):
+        model = OnnxLightGlue(weights)
+        assert model is not None
+
+    @pytest.mark.slow
+    def test_forward(self):
+        model = OnnxLightGlue()
+
+        device = torch.device("cuda")
+        kpts = torch.zeros(1, 5, 2, device=device)
+        desc = torch.zeros(1, 5, 128, device=device)
+        image = torch.zeros(1, 3, 10, 10)
+        outputs = model(
+            {
+                "image0": {"keypoints": kpts, "descriptors": desc, "image": image},
+                "image1": {"keypoints": kpts, "descriptors": desc, "image": image},
+            }
+        )
+
+        assert "matches" in outputs
+        assert "scores" in outputs
+
+    @pytest.mark.slow
+    def test_exception(self):
+        with pytest.raises(RuntimeError):
+            OnnxLightGlue(device="invalid device")
```

Comment:
[can you add assert to error messages?](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.md#coding-standards)
```suggestion
        with pytest.raises(RuntimeError) as errinf:
            OnnxLightGlue(device="invalid device")
        assert 'you error message here' in str(errinf)
```